grails-app/i18n/messages.properties

```diff
@@ -2459,6 +2459,7 @@ stockMovement.monthlyQuantity.label = Monthly stocklist qty
 stockMovement.sortByBins.label = Sort by bins
 stockMovement.originalOrder.label = Original order
 stockMovement.saveAndExit.label = Save and exit
+stockMovement.ReplenishmentPeriodNeeded.label = Replenishment period needed

```

Comment:
Maybe we should to stick to the convention and start with a lowercase letter? 
stockMovement.replenishmentPeriodNeeded.label